test/fixtures/output/kafka/client-config/create/csharp-help.golden

```diff
@@ -8,7 +8,7 @@ Create a C# client configuration file.
 
   $ confluent kafka client-config create csharp
 
-Create a C# client configuration file with arguments passed via flags.
+Create a C# client configuration file with arguments passed with flags.
```

Comment:
This description is basically just describing how a CLI command works...
```suggestion
Create a C# client configuration file with API key "my-key".
```

Reply:
I think instead I just want to remove the highlighted part completely, since a whole bunch of different examples are auto-generated here and it seems kind of weird to single out just one flag in the description.